### PR TITLE
[Cadl APIView] Release cadl-apiview 0.3.1

### PR DESCRIPTION
**Pull request by tjprescott**
Compatible with cadl-lang 0.38.0 and azure-tools 0.24.0

### CODE REVIEW

File: tools/apiview/emitters/cadl-apiview/src/emitter.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // See: https://cadlwebsite.z1.web.core.windows.net/docs/extending-cadl/emitters-basics
```
Comment (timotheeguerin) [inline tools/apiview/emitters/cadl-apiview/src/emitter.ts:1]: Do you have no change to do to handle scalars?

File: tools/apiview/emitters/cadl-apiview/test/test-host.ts [hunk 2]
```diff
@@ -15,14 +15,17 @@ export async function createApiViewTestRunner({
...
+    "Cadl.Rest",
+    "Cadl.Http",
+  ]
+  if (withVersioning) {
+    autoUsings.push("Cadl.Versioning");
+  }
+  return createTestWrapper(host, {
+    autoUsings: autoUsings,
+    compilerOptions: {
+      emitters: { "@azure-tools/cadl-apiview": {} },
```
Comment (timotheeguerin) [inline tools/apiview/emitters/cadl-apiview/test/test-host.ts:28]:
```suggestion
      emit: ["@azure-tools/cadl-apiview"],
```

`emitters` is deprecated

File: tools/apiview/emitters/cadl-apiview/src/emitter.ts (hunk 1 above)
Comment (tjprescott) [reply tools/apiview/emitters/cadl-apiview/src/emitter.ts:1]: I will now 😄  thanks for the reminder!

**Review by johanste — APPROVED** [review]

File: tools/apiview/emitters/cadl-apiview/src/emitter.ts (hunk 1 above)
Comment (tjprescott) [reply tools/apiview/emitters/cadl-apiview/src/emitter.ts:1]: Added

**Pull request merged**